castle/cms/profiles/default/controlpanel.xml

```diff
@@ -105,4 +105,17 @@
   <permission>Plone Site Setup: Users and Groups</permission>
  </configlet>
 
+
+   <configlet
+       title="Status"
+       action_id="status-controlpanel"
+       appId="status-controlpanel"
+       category="Products"
+       condition_expr=""
+       icon_expr=""
```

Comment:
Might be worth contacting @RobZoneNet about creating an icon for this panel

Reply:
@RobZoneNet could you share with Matt those URLs for icon choices?